### PR TITLE
updated .pylintrc file and added link to documentation in readme file

### PR DESCRIPTION
**Pull request by maksym-derkach**
Dear colleagues,

Please review and approve PR for the task #75 
I optimized '.pylintrc' file and added documentation on project wiki page

Thank you!

### CODE REVIEW

File: .pylintrc [hunk 1]
```diff
@@ -1,650 +1,49 @@
...
-
-# Allow loading of arbitrary C extensions. Extensions are imported into the
-# active Python interpreter and may run arbitrary code.
-unsafe-load-any-extension=no
-
-# In verbose mode, extra non-checker-related info will be displayed.
-#verbose=
-
+# Skip specific directories (Project Requirement)
+ignore=migrations,tests
```
Comment (mehalyna) [inline .pylintrc:6]: Ignoring the `tests` directory can allow poorly written tests to go unnoticed.
Remove `tests` from the ignore list and ensure test files adhere to linting rules. If there are specific test-related issues, suppress only those warnings.

File: .pylintrc [hunk 2]
```diff
@@ -1,650 +1,49 @@
...
-check-protected-access-in-special-methods=no
-
-# List of method names used to declare (i.e. assign) instance attributes.
-defining-attr-methods=__init__,
-                      __new__,
-                      setUp,
-                      asyncSetUp,
-                      __post_init__
+# Avoid these variable names (Recommended Best Practice)
+bad-names=foo,bar,baz,toto,tutu,tata
```
Comment (mehalyna) [inline .pylintrc:20]: The list of bad variable names is limited and doesn't cover common meaningless names.
Expand the bad-names list, e.g.:
```
bad-names=foo,bar,baz,toto,tutu,tata,tmp,temp,var,data,test
```

File: .pylintrc [hunk 3]
```diff
@@ -1,650 +1,49 @@
...
-# In verbose mode, extra non-checker-related info will be displayed.
-#verbose=
-
+# Skip specific directories (Project Requirement)
+ignore=migrations,tests
 
 [BASIC]
-
-# Naming style matching correct argument names.
+# Enforce naming conventions (Recommended Best Practice)
```
Comment (mehalyna) [inline .pylintrc:9]: Django views often use arguments like `self`, `request`, or `kwargs`, which can trigger "unused argument" warnings.
Add these to the ignored argument names:
```
[BASIC]
good-names=self,cls,request,kwargs,queryset
```

File: .pylintrc [hunk 4]
```diff
@@ -1,650 +1,49 @@
...
-msg-template=
-
-# Set the output format. Available formats are: text, parseable, colorized,
-# json2 (improved json format), json (old json format) and msvs (visual
-# studio). You can also give a reporter class, e.g.
-# mypackage.mymodule.MyReporterClass.
-#output-format=
-
-# Tells whether to display a full report or only the messages.
+# Simplify reporting (Recommended Best Practice)
```
Comment (mehalyna) [inline .pylintrc:45]: The `reports=no` setting disables detailed reports, which can be useful for debugging during early development stages.

**Review by mehalyna — CHANGES_REQUESTED** [review]

**Review by djmelyk-debelyk — APPROVED** [review]

File: .pylintrc [hunk 5]
```diff
@@ -1,650 +1,49 @@
...
 
-# List of valid names for the first argument in a metaclass class method.
-valid-metaclass-classmethod-first-arg=mcs
+# Define indentation (Recommended Best Practice)
+indent-string='    '
 
+[TYPECHECK]
+# Ignore Django-specific modules and classes (Project Requirement)
+ignored-modules=django.http, django.db
+ignored-classes=django.db.models
```
Comment (olexiyko) [inline .pylintrc:32]: You can also expand this list with this modules but its not necessary
**ignored-modules=django.http,django.db,django.forms
ignored-classes=django.db.models,rest_framework.serializers**

File: .pylintrc [hunk 6]
```diff
@@ -1,650 +1,49 @@
...
-# printed.
-never-returning-functions=sys.exit,argparse.parse_error
-
-# Let 'consider-using-join' be raised when the separator to join on would be
-# non-empty (resulting in expected fixes of the type: ``"- " + " -
-# ".join(items)``)
-suggest-join-with-non-empty-separator=yes
-
+# Suppress specific warnings (Project Requirement)
+disable=missing-docstring,no-member,invalid-name
```
Comment (olexiyko) [inline .pylintrc:42]: Also you can expand this list with this items:
``` disable=missing-docstring,no-member,invalid-name,too-few-public-methods,abstract-method,attribute-defined-outside-init ```

**Review by olexiyko — APPROVED** [review]
I added two comments but they are not critical

File: .pylintrc (hunk 6 above)
Comment (maksym-derkach) [reply .pylintrc:42]: The proposed list was added

File: .pylintrc (hunk 5 above)
Comment (maksym-derkach) [reply .pylintrc:32]: also were added

File: .pylintrc (hunk 4 above)
Comment (maksym-derkach) [reply .pylintrc:45]: changed reports=yes, thanks

File: .pylintrc (hunk 3 above)
Comment (maksym-derkach) [reply .pylintrc:9]: added proposed 'good-names', thanks

File: .pylintrc (hunk 2 above)
Comment (maksym-derkach) [reply .pylintrc:20]: updated bad-names

File: .pylintrc (hunk 1 above)
Comment (maksym-derkach) [reply .pylintrc:6]: the 'tests' were deleted from the ignore list

**Review by maksym-derkach — COMMENTED** [review]
all proposed changes were added

**Review by mehalyna — APPROVED** [review]

**Pull request merged**